Content.Shared/Pinpointer/SharedNavMapSystem.cs

```diff
@@ -208,14 +240,14 @@ private void OnGetState(EntityUid uid, NavMapComponent component, ref ComponentG
     [Serializable, NetSerializable]
     protected sealed class NavMapComponentState : ComponentState, IComponentDeltaState
     {
-        public Dictionary<(NavMapChunkType, Vector2i), Dictionary<AtmosDirection, ushort>> Chunks = new();
+        public Dictionary<Vector2i, Dictionary<AtmosDirection, ushort>?[]> Chunks = new();
```

Comment:
AFAICT the `Dictionary<AtmosDirection, ushort>?[]` could just be a multidim array.

Reply:
Yeah I made a separate issue for that one, if it's only ever 4 atmos directions (uhh unless z-levels do something) you can just make it a 4x array instead of the dictionary.